elastic/datadog_checks/elastic/elastic.py

```diff
@@ -380,6 +391,37 @@ def _process_policy_data(self, data, version, base_tags):
             for metric, desc in iteritems(slm_stats):
                 self._process_metric(policy_data, metric, *desc, tags=tags)
 
+    def _process_cat_allocation_data(self, admin_forwarder, version, base_tags):
+        CAT_ALLOC_PATH = '/_cat/allocation?v=true&format=json&bytes=b'
+        cat_allocation_url = self._join_url(CAT_ALLOC_PATH, admin_forwarder)
+        cat_allocation_data = self._get_data(cat_allocation_url)
+        cat_allocation_metrics = {}
+        cat_allocation_metrics.update(CAT_ALLOCATION_METRICS)
+
+        # we need to remap metric names because the ones from elastic
+        # contain dots and that would confuse `_process_metric()` (sic)
+        for dic in cat_allocation_data:
+            json.dump(dic, sys.stdout)
```

Comment:
Hey @hithwen 👋 At the moment, running `ddev validate metadata` shows the following error `elastic:25 `elasticsearch.disk.indices `contains unicode characters`. From what I can see, this is because the dictionary returned by `Elastic` has unicode characters. Any ideas how to fix that? That's my first PR so don't hesitate to ping me or/and let me know if there's a process to follow! Thank you!